TypeCobol/Compiler/Directives/CompilerDirective.cs

```diff
@@ -455,7 +453,7 @@ public CopyDirective(CompilerDirectiveType type, Token copyOrExecToken) : base(t
         public bool InsertSuffixChar { get; set; }
```

Comment:
Optional : could be changed to :
```suggestion
        public bool InsertSuffixChar => Suffix != null;
```

Reply:
It works, it relies on the fact that the `Suffix` property of `CopyDirective` is written only in case of a variation having a suffix.